### ISSUE TITLE
Add ariapressed statement to input group glyph

### ISSUE THREAD

**Issue by AlexanderSchygulaEviden** [SAP/fundamental-ngx#12958]
#### Is this a bug, enhancement, or feature request?
enhancement
#### Describe your proposal.
The aria glyph or the corresponding button of an input group should be able to be anounced "Pressed" by screenreaders, for this the interace has to accept an attribute like "ariaGlyphPressed" and take an boolean that will then be written into the attr.aria-pressed attribute.

#### Can you handle that on the application side
no

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular 17
ngx fundamental 50 and 54.1 (still the case in most current version)
#### Did you check the documentation and the API?
yes
#### Did you search for similar issues?
no
